blueprints.yaml

```diff
@@ -23,66 +23,81 @@ form:
         0: Disabled
       validate:
         type: bool
-    content_message:
-      type: text
-      label: Message
-      help: Message on the banner
-      placeholder: "This website uses cookies to ensure you get the best experience on our website."
+    assets:
+      type: section
+      title: Assets
+      underline: true
+    built_in_css:
```

Comment:
I am wunder whats that option is all about.
Look like when is option is turned off no JS and CSS is added and no template is rendered. basically its looks like disabling the hole plugin but with a extra option.

```
        // Load cookie consent JS if enabled
        if (!$this->isAdmin() && $builtInJs) {
          // Add cookie consent JS main script
          $assets->addJs('//cdnjs.cloudflare.com/ajax/libs/cookieconsent2/3.0.3/cookieconsent.min.js');
          // Add cookie consent JS settings
          $jsSettings = $twig->twig->render('partials/cookieconsent.html.twig', $cookieConsentOptions);
          $group = $loadJsOptionsBottom ? 'bottom' : null;
          $assets->addInlineJs($jsSettings, null, $group);
        }
```

Reply:
Because some people use bundle bundler like webpack to optimise loading performance.
This is what I do, I import on my own the js and css with NPM and bundle them with the rest of my assets.

The only thing that's needed in that case is the JS settings.

Reply:
My goal was a plug and play solution. Install the plugin. Maybe change colors, position and text over the admin interface and your good to go. No coding required.

If you have your own theme, using webpack to bundle cookieconsent assets, loading the cookieconsent over a customer event - there is not much left for the plugin to do. And if you disable `buildInCss` and `buildInJs` the plugin actually do nothing at all. I don't see the point in installing the plugin in this case.